common/src/main/java/org/apache/sedona/common/utils/RasterUtils.java

```diff
@@ -95,7 +100,7 @@ public static AffineTransform2D getAffineTransform(GridCoverage2D raster, PixelO
         return (AffineTransform2D) crsTransform;
     }
 
-    public static Point2D getCornerCoordinates(GridCoverage2D raster, int colX, int rowY) throws TransformException {
+    public static Point2D getWorldCornerCoordinates(GridCoverage2D raster, int colX, int rowY) throws TransformException {
```

Comment:
The problem might not be related to this PR. However, I think only `getWorldCornerCoordinatesWithRangeCheck` should be kept. I understand some PostGIS flavors do not perform range check. Do we actually support the computation of out of boundary raster coordinate?